debian/rules

```diff
@@ -59,6 +59,7 @@ override_dh_installinit:
 	install -d -m0755 debian/ceph-osd/etc/init
 	install -m0644 src/upstart/ceph-osd*.conf debian/ceph-osd/etc/init
 	install -m0644 src/upstart/ceph-disk.conf debian/ceph-osd/etc/init
+	install -m0644 build/src/upstart/ceph-osd.conf debian/ceph-osd/etc/init
```

Comment:
@johnlinp i think we can still have it, as the systemd will stay with us in the foreseeable future.